[fix] py: sxng wheel build still broken

leaty opened the pull request:
Fixes sxng wheel build properly (related PR https://github.com/searxng/searxng/pull/5510 and issue #5509), the following files were not being included:
```
static/themes/simple/chunk/
├── 13gvpunf.min.js
├── 13gvpunf.min.js.map
├── BAcZkB_P.min.js
├── BAcZkB_P.min.js.map
├── CHkLfdMs.min.js
├── CHkLfdMs.min.js.map
├── CyyZ9XJS.min.js
├── CyyZ9XJS.min.js.map
├── Db5v-hxx.min.js
├── Db5v-hxx.min.js.map
├── DBO1tjH7.min.js
├── DBO1tjH7.min.js.map
├── DxJxX49r.min.js
├── DxJxX49r.min.js.map
├── gZqIRpW1.min.js
├── gZqIRpW1.min.js.map
├── KPZlR0ib.min.js
├── KPZlR0ib.min.js.map
├── Q2SRo2ED.min.js
└── Q2SRo2ED.min.js.map
static/themes/simple/img/
├── empty_favicon.svg
├── favicon.png
├── favicon.svg
├── img_load_error.svg
├── searxng.png
├── searxng.svg
├── select-dark.svg
└── select-light.svg
````

It looked kinda clean without a logo though honestly :+1: 

<img width="1258" height="683" alt="screen 2025-12-03 12 46 38" src="https://github.com/user-attachments/assets/90696c61-5138-46aa-b220-2a895897c846" />


### Note
You could use [globbing](https://github.com/pypa/setuptools/pull/3309) since [setuptools v62.3.0](https://setuptools.pypa.io/en/stable/history.html#v62-3-0) to avoid this kind of issue, but I didn't want to potentially break compat (if anyone really needs older setuptools).

E.g.
```py
'static/themes/**/*',
'templates/**/*',
'translations/**/*',
```

leaty committed:
[fix] py: sxng wheel build still broken

Directories chunk and img were not included

Signed-off-by: leaty <dev@leaty.net>

HLFH commented:
@Bnyro Hi. Can this be merged? Thanks!

@leaty By the way, I have updated the `searxng-git` AUR package (I am the maintainer).

I have:
 - applied your patch (of this PR), it works flawlessly ;
 - removed `uwsgi` dependency ;
 - added systemd unit `searxng.service` that uses `searxng-run` executable ;
 - optimized the `python` virtual environment.

return42 committed:
[mod] setup.py package_data - use recursive globs for package_data

To test this patch build a python wheel::

    $ make clean py.build

and llok out if you are missing any files in the wheel::

    $ unzip -l dist/searxng-*-py3-none-any.whl

Signed-off-by: Markus Heiser <markus.heiser@darmarit.de>
